builds/fabric8-che/plugins/keycloak-plugin-token-provider/src/main/java/com/redhat/che/keycloak/token/provider/service/KeycloakTokenProvider.java

```diff
@@ -50,23 +50,23 @@ public KeycloakTokenProvider(@Nullable @Named("che.keycloak.github.endpoint") St
 
     /**
      * Return GitHub access token based on Keycloak token
-     * 
+     *
      * <p>
-     * Note: valid response from keycloak endpoint: 
+     * Note: valid response from keycloak endpoint:
      *       access_token=token&scope=scope&token_type=bearer
      * </p>
-     * 
+     *
      * @param keycloakToken
-     * 
-     * @return GitHub access token 
-     * @throws IOException 
-     * @throws BadRequestException 
-     * @throws ConflictException 
-     * @throws NotFoundException 
-     * @throws ForbiddenException 
-     * @throws UnauthorizedException 
-     * @throws ServerException 
-     * 
+     *
+     * @return GitHub access token
+     * @throws IOException
+     * @throws BadRequestException
+     * @throws ConflictException
+     * @throws NotFoundException
+     * @throws ForbiddenException
+     * @throws UnauthorizedException
+     * @throws ServerException
+     *
```

Comment:
This is an exact comparison with the string expected from the endpoint, whereas `Boolean.parseBoolean()` ignores case.